### ISSUE TITLE
On Provenance, dynamically loaded JS 404s

### ISSUE THREAD

**Issue by mccalluc** [refinery-platform/refinery-platform#2341]
* Specific code commit: 09a2aa2e7e739fb286f370258e899dfe27f5d9c5
* Version of the web browser and OS: Any
* Environment where the error occurred: Prod test

### Steps to reproduce
1. Try to bring up the Provenance tab

### Observed behavior

It just spins: JS console reports a syntax error on unexpected '<'. Network logs show lots of 404s:
```
https://test.stemcellcommons.org/static/js/
.../refinery/contents.js?_=1510159952098
.../provis/provvis-helpers.js?_=1510159952099
.../provis/provvis-decl.js?_=1510159952100
.../provis/provvis-init.js?_=1510159952101
.../provis/provvis-layout.js?_=1510159952102
.../provis/provvis-motifs.js?_=1510159952103
.../provis/provvis-render.js?_=1510159952104
```

We should be getting static assets from s3, so maybe the helper method isn't being used here?

### Expected behavior
* Provenance loads

**Comment by jkmarx** [comment]
I know the issue. 'static' url was hardcoded and not using the dynamically generate method. Will create a hot fix. @mccalluc

**Comment by scottx611x** [comment]
Fixed by #2347
